platform/pulp/platform/models/repository.py

```diff
@@ -1,10 +1,268 @@
 """
-Django models related to repositories.
+Repository related Django models.
 """
+from datetime import datetime
 
-from pulp.platform.models import Model
+from django.db import models
+from django.contrib.contenttypes import fields
 
+from pulp.platform.models import Model, Notes, Scratchpad, Config
 
-class Distributor(Model):
+
+class Content(Model):
     # placeholder
-    pass
+    type = models.TextField()
```

Comment:
It's also worth pointing out that in an ideal (python 3) world, python has recognized the folly of polluting the global namespace with all of its builtin functions. If you really need the builtin type in this local namespace, `import builtins; builtins.type`. `six` provides a compatibility layer to get builtins from the correct place if we want to run on py2 or py3 (but please let's just shoot for ultimately being py3)